hoomd_organics/base/system.py

```diff
@@ -209,15 +314,22 @@ def hoomd_forcefield(self):
 
     @property
     def target_box(self):
+        """The target box size of the system in form of a numpy array.
+
```

Comment:
We could add a short note about how this property can be passed into  the `hoomd_orgaics.base.Simulation.run_update_volume` method to reach the target density.